ncbi_genome_download/core.py

```diff
@@ -185,56 +252,98 @@ def download(**kwargs):
         to use multiprocessing for requests
     table : str
         file to store metadata
+    Returns
+    -------
+    int
+        success code
+    """
+    # Parse and pre-process keyword arguments
+    parser=argument_parser()
+    group = kwargs.pop('group', EDefaults.TAXONOMIC_GROUPS.default)
+    args = parser.parse_args([group])
```

Comment:
That seems like a weird hack just to get hold of an `argparse.Namespace` object.

Reply:
You are right. This is not necessary.